lte/gateway/python/magma/enodebd/data_models/data_model.py

```diff
@@ -102,7 +102,7 @@ def get_present_params(self) -> List[ParameterName]:
     def get_names_of_optional_params(cls) -> List[ParameterName]:
         all_optional_params = []
         for name in cls.get_parameter_names():
-            if cls.get_parameter(name).is_optional:
+            if cls.get_parameter(name) and cls.get_parameter(name).is_optional:  # type: ignore
```

Comment:
ignored since none check was added but mypy still can't infer the correct type in the function call (extracting variable would work but makes the code unnecessarily hard to read)